plutus-tx-plugin/test/Plugin/Primitives/string.plc.golden

```diff
@@ -1,23 +1 @@
-(program
-  (let
-    (rec)
-    (datatypebind
-      (datatype
-        (tyvardecl List (fun (type) (type)))
-        (tyvardecl a (type))
-        Nil_match
-        (vardecl Nil [List a]) (vardecl Cons (fun a (fun [List a] [List a])))
-      )
-    )
-    [
-      [ { Cons (con char) } (con char 't') ]
-      [
-        [ { Cons (con char) } (con char 'e') ]
-        [
-          [ { Cons (con char) } (con char 's') ]
-          [ [ { Cons (con char) } (con char 't') ] { Nil (con char) } ]
-        ]
-      ]
-    ]
-  )
-)
\ No newline at end of file
+(program (con string "text"))
```

Comment:
Is this right?

Reply:
Looks right with the new code. Just not sure why the old golden file is much longer.

Reply:
Because the old one was a very complicated term which represented a list of characters, this one is a nice simple constant!